src/sonic-config-engine/minigraph.py

```diff
@@ -340,11 +340,15 @@ def parse_dpg(dpg, hname):
             vintfname = vintf.find(str(QName(ns, "Name"))).text
             vlanid = vintf.find(str(QName(ns, "VlanID"))).text
             vintfmbr = vintf.find(str(QName(ns, "AttachTo"))).text
+            vlantype = vintf.find(str(QName(ns, "Type"))).text
             vmbr_list = vintfmbr.split(';')
             for i, member in enumerate(vmbr_list):
                 vmbr_list[i] = port_alias_map.get(member, member)
                 sonic_vlan_member_name = "Vlan%s" % (vlanid)
-                vlan_members[(sonic_vlan_member_name, vmbr_list[i])] = {'tagging_mode': 'untagged'}
+                if vlantype == "UserDefinedL2Vlan":
```

Comment:
what is UserDefinedL2Vlan?

Reply:
Thanks for reviewing! This is just a keyword we have created internally within our team to denote a tagged vlan type. Hopefully this is an ok naming convention?

Reply:
I have also added the UT. Please let me know if it looks ok.